integration-tests/go.mod

```diff
@@ -8,8 +8,8 @@ require (
 	github.com/onsi/gomega v1.19.0
 	github.com/rs/zerolog v1.27.0
 	github.com/satori/go.uuid v1.2.0
-	github.com/smartcontractkit/chainlink-env v0.2.18
-	github.com/smartcontractkit/chainlink-testing-framework v1.4.2
+	github.com/smartcontractkit/chainlink-env v0.2.19
+	github.com/smartcontractkit/chainlink-testing-framework v1.4.3-0.20220701101352-08f4f955cdf4
```

Comment:
Is it possible to stick to a tagged semver release instead?

Reply:
The changes made in this PR are dependent on some new additions in the `chainlink-testing-framework` repo (the ones included in the has of the commit indicated on line 12), so we'd need a new release from the QA team, which I don't know if they would do just for these changes. Is it bad practice to point to a commit like I do at the moment?

Reply:
It is not necessarily problematic, but it is harder for us to track as developers and may overly constrain how the tooling resolves the versions. But regardless, It looks like github.com/smartcontractkit/chainlink-testing-framework is on 1.4.6 now, no? Shouldn't this either be included in a tagged version, or a variant of the latest version?

Reply:
I think this is expected since we are doing these changes often in testing framework. 
Not sure @kalverra if you have any better solution in mind?

Reply:
Apologies Jordan, I wasn't sure when you're going to reply and I merged the PR before I saw your reply...I understand, that makes sense, thank you for letting me know! I am going to do a refactoring anyway today for the Keeper tests, so I can change the version then to a proper one. Would that be all right? In a couple of hours or so.

Reply:
No bother - sergey actually already has another bump in flight here: #6909

Reply:
Oh great, that's even better then! Thanks!